Dialog/CreditEditor_TableView.cpp

```diff
@@ -0,0 +1,237 @@
+                    this->model()->setData(model_index,QColor("#8080ff"), Qt::BackgroundRole);
+                }
+
+                //Selecting next index
+                if (index_list.size() == 1) {
+                    QModelIndex model_index=index_list.at(0);
+                    int column=model_index.column();
+                    int row=model_index.row();
+                    column++;
+
+                    QModelIndex newindex=this->model()->index(row,column);
+                    this->selectionModel()->select(newindex,QItemSelectionModel::SelectCurrent);
+
+                }
+            }
+        }
+    } else if (keycode == Qt::Key_Left) {
+        QModelIndexList index_list=this->selectionModel()->selectedIndexes();
+        if (index_list.size() == 1) {
+            QModelIndex model_index=index_list.at(0);
+            int column=model_index.column();
+            int row=model_index.row();
+            column--;
+
+            if (column >= 0) {
+                QModelIndex newindex=this->model()->index(row,column);
+                this->selectionModel()->select(newindex,QItemSelectionModel::SelectCurrent);
+            }
+        }
+    } else if (keycode == Qt::Key_Right || Qt::Key_Space) {
+        QModelIndexList index_list=this->selectionModel()->selectedIndexes();
+        if (index_list.size() == 1) {
+            QModelIndex model_index=index_list.at(0);
+            int column=model_index.column();
+            int row=model_index.row();
+            column++;
+
+            if (column < 32) {
+                QModelIndex newindex=this->model()->index(row,column);
+                this->selectionModel()->select(newindex,QItemSelectionModel::SelectCurrent);
+            }
+        }
+    } else if (keycode == Qt::Key_Up) {
+        QModelIndexList index_list=this->selectionModel()->selectedIndexes();
+        if (index_list.size() == 1) {
+            QModelIndex model_index=index_list.at(0);
+            int column=model_index.column();
+            int row=model_index.row();
+            row--;
+
+            if (row >= 0) {
+                QModelIndex newindex=this->model()->index(row,column);
+                this->selectionModel()->select(newindex,QItemSelectionModel::SelectCurrent);
+            }
+        }
+    } else if (keycode == Qt::Key_Down) {
+        QModelIndexList index_list=this->selectionModel()->selectedIndexes();
+        if (index_list.size() == 1) {
+            QModelIndex model_index=index_list.at(0);
+            int column=model_index.column();
+            int row=model_index.row();
+            row++;
+
+            if (row < 20) {
+                QModelIndex newindex=this->model()->index(row,column);
+                this->selectionModel()->select(newindex,QItemSelectionModel::SelectCurrent);
+            }
+        }
+    }
+}
+
+
+/// <summary>
+/// Delete function used (called by delete key or right click->delete)
+/// Delete is consistent across theme
```

Comment:
Not a change request, but I am wondering why this distinction is made. Are there other actions which have proven to be inconsistent across themes?

Reply:
@KleymanStuff

Reply:
Don't know, it seems to be only the case for object involving a default color.